engine/src/main/java/org/camunda/bpm/engine/history/HistoricVariableUpdate.java

```diff
@@ -60,4 +60,9 @@
    * @return an error message indicating why the variable value could not be loaded.
    */
   String getErrorMessage();
+
+  /**
+   * @return true if the detail historic variable update is created during the process instance start
+   */
+  boolean isInitial();
```

Comment:
I think this should be the wrapper `Boolean` class to handle cases with `null` properly? Same for the field declaration in the entity. Or are the database fields in all cases declared to not allow null and have a proper default value (e.g. {{INITIAL_ smallint check(INITIAL_ in (1,0));}} sounds like it)? In that case we must make sure that rolling-update scenarios still work, i.e. a 7.12 engine creates a history detail in the 7.13 schema.